Xwt.Gtk/Xwt.GtkBackend/BoxBackend.cs

```diff
@@ -150,6 +150,13 @@ protected override void OnSizeAllocated (Gdk.Rectangle allocation)
 				cr.Key.SizeAllocate (new Gdk.Rectangle (allocation.X + (int)r.X, allocation.Y + (int)r.Y, (int)r.Width, (int)r.Height));
 			}
 		}
+
+		protected override void OnSizeRequested(ref Gtk.Requisition requisition)
```

Comment:
This breaks the Gtk3 build and should go into https://github.com/mono/xwt/blob/xwt-siz-request/Xwt.Gtk/Xwt.GtkBackend/BoxBackendGtk2.cs.

Reply:
:+1: